admin/templates/nodes/node.html

```diff
@@ -15,93 +15,92 @@
 {% block content %}
     <div class="container-fluid">
         <div class="row">
-            <a href="{% url 'nodes:search' %}"
-               class="btn btn-primary col-md-1">
-                <i class="fa fa-search"></i>
-            </a>
-            <a href="{% url 'nodes:node-logs' node.id %}" class="btn btn-primary col-md-1">
-                View Logs
-            </a>
-            {%  if perms.osf.delete_node %}
-                {% if not node.is_registration %}
-                    {% if not node.deleted %}
-                        <span class="col-md-1">
-                        <a href="{% url 'nodes:remove' guid=node.id %}"
-                           data-toggle="modal" data-target="#deleteModal"
-                           class="btn btn-danger">
-                            Delete Node
+            <div class="col-md-12">
+                <div class="btn-group" role="group">
+                    <a href="{% url 'nodes:search' %}" class="btn btn-primary"> <i class="fa fa-search"></i></a>
+                    <a href="{% url 'nodes:node-logs' node.id %}" class="btn btn-primary">View Logs</a>
+                    {%  if perms.osf.delete_node %}
+                        {% if not node.is_registration %}
+                            {% if not node.deleted %}
+                                <a href="{% url 'nodes:remove' guid=node.id %}"
+                                   data-toggle="modal" data-target="#deleteModal"
+                                   class="btn btn-danger">
+                                    Delete Node
+                                </a>
+                                <div class="modal" id="deleteModal">
+                                    <div class="modal-dialog">
+                                        <div class="modal-content"></div>
+                                        {# Data from above link #}
+                                    </div>
+                                </div>
+                            {% else %}
+                                <form method="post"
+                                      action="{% url 'nodes:restore' guid=node.id %}">
+                                    {% csrf_token %}
+                                    <input class="btn btn-success" type="submit"
+                                           value="Restore Node" />
+                                </form>
+                            {% endif %}
+                        {% else %}
+                            <a href="{% url 'nodes:restart-stuck-registrations' guid=node.id %}"
+                               data-toggle="modal" data-target="#confirmRestartRegistration"
+                               class="btn btn-warning">
+                                Restart Registration 
```

Comment:
It looks like we give the ability to Restart Registration for every single registration.  The force_archive script does verifications that the force_archive can occur, so maybe this shouldn't be an option for all.